lib/qmp.ml

```diff
@@ -29,11 +29,20 @@ type enabled = {
   present: bool;
 }
 
+type vnc = {
+  enabled : bool;
+  auth    : string;
+  family  : string;
+  service : string;
```

Comment:
I think it would be nice to add some validation here, like checking that `family` and `host` are meaningful, `auth` makes sense and service is an `int` (is it?). But probably it's overkill for this module.